modules/wpcom-block-editor/class-jetpack-wpcom-block-editor.php

```diff
@@ -66,9 +67,14 @@ public function is_iframed_block_editor() {
 	 */
 	public function show_error_if_logged_out() {
 		if ( ! get_current_user_id() ) {
-			$login_url = wp_login_url();
 			/* translators: %s: Login URL */
-			wp_die( sprintf( __( 'In order to use the block editor of this Jetpack site in WordPress.com, you need to <a href="%s" target="_blank" rel="noopener noreferrer">log into</a> the Jetpack site.', 'jetpack' ), $login_url ) ); // // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped
+			$message = __( 'Please <a href="%s" target="_blank" rel="noopener noreferrer">log into</a> your Jetpack-connected site to use the block editor on WordPress.com.', 'jetpack' );
```

Comment:
@Automattic/editorial I wonder if this error message is a bit curt? Any suggestions here would be appreciated!